docs/guides/multiple_deployment_tracks.md

```diff
@@ -0,0 +1,245 @@
+
+:::info
+
+- The `internal` variant should only be used for internal testing/validation.
+- The `stable` variant should be shipped to end users in production.
+  :::
+
+## Creating a patch
+
+Now that we have our internal and stable releases on the Play Store, we can create a patch using `shorebird patch`. For the sake of this example, let's adjust the app theme to use `deepOrange` as the seed color in `lib/main.dart`:
+
+```diff
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      theme: ThemeData(
+        // This is the theme of your application.
+        //
+        // TRY THIS: Try running your application with "flutter run". You'll see
+        // the application has a blue toolbar. Then, without quitting the app,
+        // try changing the seedColor in the colorScheme below to Colors.green
+        // and then invoke "hot reload" (save your changes or press the "hot
+        // reload" button in a Flutter-supported IDE, or press "r" if you used
+        // the command line to start the app).
+        //
+        // Notice that the counter didn't reset back to zero; the application
+        // state is not lost during the reload. To reset the state, use hot
+        // restart instead.
+        //
+        // This works for code too, not just values: Most code changes can be
+        // tested with just a hot reload.
+-        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
++        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepOrange),
+        useMaterial3: true,
+      ),
+      home: const MyHomePage(title: 'Flutter Demo Home Page'),
+    );
+  }
+}
+```
+
+:::info
+Typically `shorebird patch` should be used to fix critical bugs.
+:::
+
+Now that we've applied the changes, let's patch the `internal` variant:
+
+```sh
+shorebird patch --flavor internal
+```
+
+:::tip
+Run `shorebird apps list` to verify the patch was published:
+
+```sh
+ Apps
+┌──────────────────────────────┬──────────────────────────────────────┬─────────────┬───────┐
+│ Name                         │ ID                                   │ Release     │ Patch │
+├──────────────────────────────┼──────────────────────────────────────┼─────────────┼───────┤
+│ shorebird_example (internal) │ 661c2383-4fe9-43ca-8bb0-beb5959067d6 │ 1.0.0+1     │ 1     │
+├──────────────────────────────┼──────────────────────────────────────┼─────────────┼───────┤
+│ shorebird_example (stable)   │ 1db93edf-4a8a-493a-9450-bd62d06cefb1 │ 1.0.0+1     │ --    │
+└──────────────────────────────┴──────────────────────────────────────┴─────────────┴───────┘
+```
+
+:::
+
+We can validate the patch by re-launching the internal release.
+
+:::warning
+If you are testing locally, DO NOT re-run `shorebird run`, instead just re-launch the app from the device or emulator directly.
```

Comment:
Why not? :). "shorebird run always builds and runs the latest code for your app.  Which means it will run with the patched code already included, rather than showing you the released code with the patch applied."?

Not sure. Or maybe we just need to make `shorebird run` give a warning in these cases?

Reply:
Yeah I put this in so that folks can see the patches being applied. I can reword to make this "softer" like, "If you are testing locally you do not need to re-run `shorebird run`...

Reply:
It's also fine as is.  I just like to either explain why we're asking the user to do something extra (so they can understand) or take away the footgun. :)

Reply:
Yup re-worded to hopefully make it a bit more clear.